modules/nw-externalip-about.adoc

```diff
@@ -0,0 +1,184 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-externalip.adoc
+
+[id="nw-externalip-about_{context}"]
+= About external IP addresses
+
+You can allow external traffic into the cluster by defining an *external IP* configuration.
+
+The configuration for external IP address blocks is defined by a Network custom resource (CR) named `cluster`.
```

Comment:
OpenShift builds some additional functionality on top of Kubernetes' ExternalIPs.

* Administrators can restrict the use of ExternalIPs (ExternalIPPolicy)
* Services can request an ExternalIP be automatically allocated to them. ( AutoAssignExternalIPs)

Reply:
By default, only administrators can create services with an ExternalIP set. A policy must be set to allow non-administrator use of the feature.